services/web/server/src/simcore_service_webserver/users/_preferences_models.py

```diff
@@ -79,7 +79,7 @@ class CreditsWarningThresholdFrontendUserPreference(FrontendUserPreference):
 
 class WalletIndicatorVisibilityFrontendUserPreference(FrontendUserPreference):
     preference_identifier: PreferenceIdentifier = "walletIndicatorVisibility"
-    value: str | None = "always"
+    value: str | None = "warning"
```

Comment:
This will not change it for existing users. You might want a SQL query that we need to run on the deployments when this is merged in order to also change this then.

Reply:
Exactly, read the PR description ;)